api/controllers/game/subscribe.js

```diff
@@ -40,7 +40,14 @@ module.exports = function (req, res) {
       const addPlayerToGame = Game.addToCollection(game.id, 'players').members([user.id]);
       const updatePlayer = User.updateOne({ id: user.id }).set({ pNum });
 
-      return Promise.all([game, updatePlayer, addPlayerToGame]);
+      // Set game.p0 or game.p1 as requesting user's id
+      const pNumUpdate = {};
+      const pNumKey = `p${pNum}`;
+      pNumUpdate[pNumKey] = user.id;
+      game[pNumKey] = user.id;
+      const updateGame = Game.updateOne({id: game.id}).set(pNumUpdate);
```

Comment:
Same idea with leave-lobby: we're now setting p0 and p1 upon joining the lobby instead of when the game starts in the ready action. Eventually this will deprecate the need for the players collection